spec/unit/installer/analyzer_spec.rb

```diff
@@ -62,6 +62,19 @@ module Pod
         @analyzer.update_repositories
       end
 
+      it 'does not update sources if there are no dependencies' do
```

Comment:
This is a small optimization that came along with my changes. If a Podfile has no dependencies specified, then it won't bother updating any of the sources. This seems safe to me, but I'd like confirmation from the veterans on this.